electron/main-window.ts

```diff
@@ -30,7 +30,7 @@ const createMainWindow = () => {
     webPreferences: {
       nodeIntegration: true,
       nodeIntegrationInWorker: true,
-      webSecurity: false,
+      webSecurity: true,
```

Comment:
@NayamAmarshe

Reply:
Please do not approve any code changes. Everything we've done is for a reason, no need to change stuff around in the code yet.